ci/kind/kind-setup.sh

```diff
@@ -242,6 +245,44 @@ function configure_extra_networks {
   done
 }
 
+function get_bridge_id {
+  local bridge_id=$(docker network inspect kind -f {{.ID}})
+  local return_value=$?
+  if [ $return_value -eq 0 ]; then
+        echo "$bridge_id"
+    else
+        echo "Unable to get kind bridge id, Docker command failed with exit code $return_value"
+        exit 1
+    fi
+}
+
+function configure_kind_ipam_vlan_subnets {
+  if [[ -z $IPAM_VLAN_SUBNETS || -z $IPAM_VLAN_IDS ]]; then
```

Comment:
This is almost the same as `configure_vlan_subnets`. The reason why the latter has a generic name instead of something like `configure_egress_vlan_subnets` is to make it reusable for other use cases.

So you should just update `configure_vlan_subnets` to be used by flexible-ipam instead of duplicating one mutation for each feature.